routes/test_route.py

```
@@ -1,128 +1,131 @@
-from fastapi import APIRouter, status, File, UploadFile
-from fastapi.params import Depends
+from fastapi import APIRouter, status, File, UploadFile, Depends, HTTPException
 from fastapi.responses import JSONResponse
-from sqlalchemy.orm import Session
 from sqlalchemy import text
+from sqlalchemy.orm import Session
 
+from crud.schedule import create_schedule
+from crud.user import get_user_by_id
+from crud.program import get_program_by_id
 from crud.schedule import create_schedule
```

Ruff (0.8.2)

Redefinition of unused `create_schedule` from line 8

Remove definition: `create_schedule`

(F811)